Resources/views/app/partials/ticktack.html.twig

```diff
@@ -1,5 +1,12 @@
 {# Resources/views/app/partials/ticktack.html.twig #}
 
-<pre>partials/ticktack.html.twig</pre>
 {# Use `@App` to refer to the original template #}
-{{ include('@App/partials/ticktack.html.twig') }}
+{# {{ include('@App/partials/ticktack.html.twig') }} #}
+
+<div class="nav-item d-flex me-1 ticktac">
+    <div class="btn-list">
+        <a href="{{ path('timesheet_create') }}" class="modal-ajax-form btn btn-outline-white btn-icon px-sm-2">
+            <i class="text-green fas fa-circle-plus me-0 me-sm-1"></i> {{ 'Create registration'|trans }}
```

Comment:
Kimai uses translation keys like `timesheet.start`, i.e. not real sentences. For consistency, we should consider doing the same and use `timesheet.create`, say. 

The new translation should be added in a `messages.en.xlf` file in the `Resources/translations` folder (cf. https://github.com/kimai/DemoBundle/blob/main/Resources/translations/messages.en.xlf).

Reply:
Thanks - I have changed my code to use keys instead.